### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by trueadm**

### CODE REVIEW

**Review by acywatson — APPROVED** [review]

File: packages/lexical-website-new/docs/api/lexical.md [hunk 1]
```diff
@@ -3,3 +3,65 @@ sidebar_position: 1
...
+
+- `is`
+- `isBefore`
+- `getCharacterOffset`
+- `getNode`
+- `set`
+
+### `NodeSelection`
+
+> TODO
```
Comment (maasencioh) [inline packages/lexical-website-new/docs/api/lexical.md:51]: This TODO means that this code is not implemented or that the documentation is missing?

Comment (trueadm) [reply packages/lexical-website-new/docs/api/lexical.md:51]: Docs are missing, code has been implemented.

Comment (maasencioh) [reply packages/lexical-website-new/docs/api/lexical.md:51]: Thank you, I'll be pleased to help if nobody is working on this

Comment (trueadm) [reply packages/lexical-website-new/docs/api/lexical.md:51]: I don't believe anyone is, so any help would be much appreciated!

**Pull request merged**